src/stan/error_handling/matrix/check_ldlt_factor.hpp

```diff
@@ -28,9 +38,8 @@ namespace stan {
         msg << "is not positive definite. "
             << "last conditional variance is ";
```

Comment:
The old messages were often awkward to comply with Boost's
general error dumper.  I think it makes sense to print all of
the matrix or none of it.  Or maybe all of it up to a fixed
max number of entries?

> On Jan 12, 2015, at 5:45 PM, Daniel Lee <notifications@github.com> wrote:
> 
> In src/stan/error_handling/matrix/check_ldlt_factor.hpp:
> 
> > @@ -28,9 +38,8 @@ namespace stan {
> >          msg << "is not positive definite. "
> >              << "last conditional variance is ";
> 
> I don't know what this means.
> @bgoodri or @mbrubake, thoughts?
> 
> —
> Reply to this email directly or view it on GitHub.
>